string effector_name
--------------------------------
bool success